integreat_cms/cms/views/pages/page_bulk_actions.py

```diff
@@ -114,3 +114,38 @@ def post(self, request, *args, **kwargs):
 
         # Let the base view handle the redirect
         return super().post(request, *args, **kwargs)
+
+
+# pylint: disable=too-many-ancestors
+class RefreshDateView(PageBulkActionMixin, BulkActionView):
+    """Bulk action for refreshing dates."""
+
+    def post(self, request, *args, **kwargs):
+        r"""
+        Function for refreshing dates/timestamps for all translations of a page
+
+        :param request: The current request
+        :type request: ~django.http.HttpResponse
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: The redirect
+        :rtype: ~django.http.HttpResponseRedirect
+        """
+        # Get all page translations to the corresponding pages of the given queryset
+        page_translations = PageTranslation.objects.filter(
+            page__id__in=self.get_queryset()
+        )
+        for page_translation in page_translations:
+            # Only refresh the date for the last version of a page translation
+            if page_translation is page_translation.latest_version:
```

Comment:
```suggestion
        for page in self.get_queryset():
            # Only refresh the date for the last version of a page translation
            for page_translation in page.translations.distinct("page__pk", "language__pk"):
```
I think this might be more performant - however, the same problem as with the individual view: We need to preserve the order of the save operations according to the region's language tree.